package.json

```diff
@@ -77,7 +77,7 @@
     "knockout": "3.4.0",
     "konva": "1.6.8",
     "magic-wand-js": "1.0.0",
-    "moment": "2.13.0",
+    "moment": "^2.22.2",
```

Comment:
Odd to see this. Why was this changed?

Reply:
I too am curious as I did not work on that file, I made a minor modification to
`interface/main/tabs/menu/menus/patient_menus/standard.json`

Reply:
I think I figured out the reason. I ran npm install on the branch by mistake. The git status did not show any changes. It is the probable cause